### PR TITLE
chore(slashcommand): general cleanup, remove debug

### PR DESCRIPTION
**Pull request by NickBurnett**

### CODE REVIEW

File: src/api/bot.ts [hunk 1]
```diff
@@ -24,6 +24,9 @@ export interface Config {
   token: string;
   dbUrl: string;
   commandPath: string;
+  slashCommandPath: string;
```
Comment (ez314) [inline src/api/bot.ts:27]: all of these paths are a bit cluttering now.. should prob move somewhere else at a later time

File: src/util/manager/interaction.ts [hunk 2]
```diff
@@ -153,7 +155,6 @@ export default class InteractionManager extends Manager {
       // Bulk update all permissions
       await guildCommandManager.permissions.set({ fullPermissions });
     } catch (error) {
-      console.log(error)
```
Comment (ez314) [inline src/util/manager/interaction.ts:156]: I should open an issue about this but too lazy rn XD

**Review by ez314 — COMMENTED** [review]
Can we remove package-lock.json tho, it makes it so u cant tell which commits are big commits

**Comment by NickBurnett** [comment]
Yes that is good no? Makes me look like I contribute more than actual xD 😳

**Review by khreate — APPROVED** [review]
LGTM!

**Comment by khreate** [comment]
Yes I 100% committed 5k changes !

**Review by ez314 — APPROVED** [review]

**Pull request merged**